### DISCUSSION TITLE
essentials+

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2292] [Q&A]
How do i install essentials+ plugin?

**Comment by Kojirremer** [comment]
Every plugin (and **only the plugin**, which is _always a DLL file_) needs to be placed on the ``ServerPlugins`` subfolder. So in your case, the plugin should be the file ``EssentialsPlus.dll``, which is the _only file_ you place on that folder.
Other plugins may have extra DLLs required, but if they do they usually are also provided, and mentioned in their **ReadMe** (which of course _you should always read_).
